members/anmolpant.md

```diff
@@ -0,0 +1,41 @@
+### Hi there, I am a developer 👋
+
+
+- 🔭 I’m currently working on getting my shit together.
+- 🔮 What do I do? - Predict the future via computer science black magic.
+- 🌱 I’m currently learning fastai for NLP
+- 👯 I’m looking to collaborate on Machine Learning and Web Development related projects.
+- 🤔 I’m looking for help with LSTMs and college assignments. 
+- 💬 Ask me about: anything but my life.
+- 📫 How to reach me: oohhh toodlesssss
+- 😄 Pronouns: Professional Disappointment
+- ⚡ Fun fact: Getting stabbed is very demotivating.
+- 📚 Words that sum me up: Music, Movies, Memes, Minimalism, Melancholy
+
+```
+                             \\\\\\\
+                            \\\\\\\\\\\\
+                          \\\\\\\\\\\\\\\
+  -----------,-|           |C>   // )\\\\|
+           ,','|          /    || ,'/////|
+---------,','  |         (,    ||   /////
+         ||    |          \\  ||||//''''|
+         ||    |           |||||||     _|
+         ||    |______      `````\____/ \
+         ||    |     ,|         _/_____/ \
+         ||  ,'    ,' |        /          |
+         ||,'    ,'   |       |         \  |
+_________|/    ,'     |      /           | |
+_____________,'      ,',_____|      |    | |
+             |     ,','      |      |    | |
+             |   ,','    ____|_____/    /  |
+             | ,','  __/ |             /   |
+_____________|','   ///_/-------------/   |
+              |===========,'
+```
+
+<div align="center">
+<img src="https://i.pinimg.com/originals/a4/f2/cb/a4f2cb80ff2ae2772e80bf30e9d78d4c.gif" width="300" height="300" />
+</div>
```

Comment:
You can remove the <div> tags for the GIF to work properly.

Reply:
Done with it?

Reply:
Yes, I have changed my readme.md file and have incorporated the above changes in my next pr.